www/js/app.js

```diff
@@ -534,6 +534,10 @@ document.querySelectorAll('input[type="checkbox"][name=hideActiveContentWarning]
         settingsStore.setItem('hideActiveContentWarning', params.hideActiveContentWarning, Infinity);
     })
 });
+document.getElementById('hideExternalLinkWarningCheck').addEventListener('change', function () {
+    params.hideExternalLinkWarning = !!this.checked;
```

Comment:
Other contributors have used `!!this.checked`, but I think it's redundant as `this.checked` alone is already truthy. It's up to you if you prefer the double negation, but I've never seen the purpose of it in this context!

Reply:
Sure, I removed it in [d0150e5](https://github.com/kiwix/kiwix-js/pull/1236/commits/d0150e5d4a86d77c7cb4b456abd49a292d5c7f24).